# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxFcPlatformFontList.h
# Commit: c03c7df7cd61
# Full Hash: c03c7df7cd61c88de130f49dbceaa8a62c9a7758
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-05-14 21:58:21
# Regressor Bug: 1708768
# File Overlap Count: 2
# Description:
#   Bug 1708768 - Preliminary cleanup of gfxPlatformFontList creation/initialization code; no functional change. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D114863
# ==============================================================================

diff -r 76dd6379cc01 -r c03c7df7cd61 gfx/thebes/gfxFcPlatformFontList.h
--- a/gfx/thebes/gfxFcPlatformFontList.h	Fri May 14 17:45:16 2021 +0000
+++ b/gfx/thebes/gfxFcPlatformFontList.h	Fri May 14 18:05:44 2021 +0000
@@ -239,7 +239,8 @@
   gfxFcPlatformFontList();
 
   static gfxFcPlatformFontList* PlatformFontList() {
-    return static_cast<gfxFcPlatformFontList*>(sPlatformFontList);
+    return static_cast<gfxFcPlatformFontList*>(
+        gfxPlatformFontList::PlatformFontList());
   }
 
   // initialize font lists